### DISCUSSION TITLE
composer error after installing this package

### DISCUSSION THREAD

**Discussion by SigalZ** [spatie/laravel-livewire-wizard#45] [Q&A]
I run Laravel 9 on Windows 10.

I just installed this package:
https://spatie.be/docs/laravel-livewire-wizard/v1/introduction

and now I get this error:
Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.1.0". You are running 8.0.13. in C:\wamp64\www\highland9.local\vendor\composer\platform_check.php on line 24

After googling it I check all of these:

php version on Wamp shows 8.1.0

Windows path points to: c:\wamp64\bin\php8.1.0

cmd php -v shows: 8.1.0

In my composer.json file:

it was:
```
"require": {
        "php": "^8.0.2",
```
I changed it to:
```
"php":"^8.1.0"
```
and ran composer update

Nothing helps, still getting this error.

How do I remove this package?

**Comment by ju5t** [comment]
This isn't an issue with this package but with your composer configuration. If you want to remove a package, it's:

```bash
composer remove vendor/package
```

**Reply by SigalZ** [reply]
Thank you, apologies, after hours of trying to find out what's wrong, it appears Wamp did not change the php version on the virtual host.

I managed to changed that and now it's working.

But, trying to remove the package did not work, I tried composer remove vendor/spatie/laravel-livewire-wizard and it said there is nothing to remove.

Ended up removing it manually.

**Reply by ju5t** [reply]
The vendor in the example should be replaced, it's 'spatie' in this case. The command is `composer remove spatie/laravel-livewire-wizard`. Always better to get it working though :)
